connect-location/src/main/java/com/ifttt/location/ConnectLocation.java

```diff
@@ -107,6 +107,23 @@ public void onError(ErrorResponse errorResponse) {
         });
     }
 
+    /**
+     * Call this method if logging is required.
+     * The Logger doesn't differentiate between release and debug build variants, so call this method accordingly.
+     * By default, logging is disabled.
+     */
+    public void enableLogging() {
```

Comment:
Maybe use one method, e.g `setLoggingEnabled(boolean)`? Probably won't matter much but still one method less from the SDK.